client/my-sites/themes/theme-upload/style.scss

```diff
@@ -1,8 +1,5 @@
 // View: placeholder with gradient
 .theme-upload {
-	.eligibility-warnings__warning {
```

Comment:
The theme screen look like this:

![image](https://user-images.githubusercontent.com/1044309/192890573-4493480c-d873-4382-969f-769e5a7863f5.png)